### DISCUSSION TITLE
How to toggle the visibility of the status bar "per-workspace"?

### DISCUSSION THREAD

**Discussion by nktnet1** [JakeStanger/ironbar#750] [Q&A] [answered]
Is there a way to show and hide the bar specific to the current (Hyprland) workspace?

E.g. supposed we hid the bar on workspace 1. Switching to workspace 2, the bar should reappear. Switching back to workspace 1, the bar should remain hidden.

---

I'm trying to simulate a small part of DWM's [pertag](https://dwm.suckless.org/patches/pertag) patch - this seems like something that might be possible with IPC/scripting, but would be nice if there's a way to do it natively.

**Comment by JakeStanger** [comment, marked as answer]
There isn't a config option for this, but you can write your own script to achieve this. You'd need to listen to Hyprland's IPC, and then talk to Ironbar's IPC to toggle visibility. (see `ironbar bar --help`).

https://wiki.hyprland.org/IPC/#how-to-use-socket2-with-bash
